free-3.md

```diff
@@ -45,6 +54,9 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are definitely a lot of calendar apps out there, some of which support more advanced scheduling features. However, many do not have this kind of flexibility and are not interactive. I think there is / was a CS research project related to an app that helped users plan their time kind of like this, but I don't know how far it got. Prediction will be key to really making something better than before, though. If this DSL could find how long it takes you to start and do homework, it could change its suggestions about when to start working on it appropriately.
```

Comment:
That project is still in development I think - it's Professor Boerkoel's project (I believe) if you want to ask him about it. It's certainly very relevant, possibly the most relevant existing project.